lib/importer.rb

```diff
@@ -8,18 +8,34 @@ def self.included(base)
     end
   end
 
+  def available_version
+    nil
+  end
+
+  def stored_version
+    model_class.stored_metadata[:version]
+  end
+
   module ParentInstanceMethods
     def import
       Rails.logger.info "#{self.class.name}: import starting."
 
-      start_time = Time.now if can_purge_old?
-      super
-      model_class.purge_old(start_time) if can_purge_old?
+      if resource_changed?
+        start_time = Time.now if can_purge_old?
+        super
+        update_metadata available_version
+        model_class.purge_old(start_time) if can_purge_old?
+      end
```

Comment:
Can you log the fact that the import was skipped due to the resource being unchanged?